ereolen_support/ereolen_support.form.inc

```diff
@@ -85,98 +77,222 @@ function ereolen_support_form($form, &$form_state) {
     '#required' => variable_get('ereolen_support_email_required')
   ];
 
+  $form['ereolen_support_date'] = [
+    '#type' => 'date_popup',
+    '#title' => t('Dato'),
+    '#date_format' => 'd/m/Y',
+    '#date_label_position' => 'none',
+    '#description' => variable_get('ereolen_support_date_description'),
+    '#required' => variable_get('ereolen_support_date_required'),
+  ];
+
+  $libraries = module_exists('publizon') ? publizon_get_libraries_list() : [];
+  $form['ereolen_support_library'] = [
+    '#type' => 'select',
+    '#title' => t('Library'),
+    '#options' => _ereolen_support_array_as_select($libraries),
+    '#description' => variable_get('ereolen_support_library_description'),
+    '#required' => variable_get('ereolen_support_library_required'),
+    '#states' => [
+      'visible' => [
+        ':input[name="ereolen_support_problem"]' => [
+          ['value' => 'Log ind'],
```

Comment:
Are these values hardcode or would it be possible to get them from the database? If the customer changes any of these in the vocab it would require a new release to fix the support forms?